.github/workflows/build.yaml

```diff
@@ -0,0 +1,23 @@
+name: Build and Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+          python-version: ['2.7', '3.5', 'pypy2.7', 'pypy3.5']
```

Comment:
I would probably just test 3.7+ or 3.8+

Reply:
I'll be dropping old python in #12. I wanted a working github actions for that PR first.

Reply:
ah -- these versions are going to fail immediately -- here's where the versions come from -- https://github.com/actions/python-versions/releases  (I'd suggest picking some of them from there)

Reply:
I don't think I need to specify minor versions. 2.7.17 and 3.5.9 are available as releases if you jump to the last page.

Reply:
ah yeah -- they'll generally resolve to the latest patch version of those -- I'm just not sure if 3.5 is going to work and I'm pretty sure the pypy ones won't work (and need to match the specific pypy versions on https://downloads.python.org/pypy/ )

Reply:
I verified `2.7` and `3.5` are valid. I verified `pypy3.5` does not exist. I'll drop `pypy3.5` here, and newer pypy will be added in #12.